src/core/cardano/walletConnect/peerConnection.ts

```diff
@@ -57,6 +57,9 @@ class PeerConnection {
   connectWithDApp(dAppIdentifier: string) {
     if (this.identityWalletConnect) {
```

Comment:
Maybe we should throw if `identityWalletConnect` is not set (i.e. start hasn't been called).